demos/classification_benchmark_demo/cpp_gapi/src/custom_kernels.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) 2022-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom_kernels.hpp"
+
+#include <algorithm>
+#include <map>
+#include <memory>
+#include <stdexcept>
+#include <utility>
+
+#include <ie_core.hpp>
```

Comment:
I think you don't need `ie` headers there

Reply:
ok